src/main/kotlin/binarysearch/Printer.kt

```diff
@@ -0,0 +1,26 @@
+package binarysearch
+
+fun main() {
+    println(solution(intArrayOf(2, 1, 3, 2), 2))
+    //println(solution(intArrayOf(1, 1, 9, 1, 1, 1), 0))
+}
+
+fun solution(priorities: IntArray, location: Int): Int {
+    var indexQueue = 0
+    var answer = 0
+    val queue = priorities.mapIndexed { index, i -> Pair(i, index) }.toMutableList()
+    val findValue = queue[location]
+    while (true) {
+        if (findValue == queue[indexQueue] && queue.maxOfOrNull { it.first } == queue[indexQueue].first) {
```

Comment:
```suggestion
    while (true) {
        if (location == queue[indexQueue].second && queue.maxOfOrNull { it.first } == queue[indexQueue].first) {
```
요소에 index를 추가하셨으니 location == queue[indexQueue].second 로 두셔도 될 것 같네용 :)

Reply:
좋습니당 :)